system-test/bigtable.ts

```diff
@@ -1259,6 +1259,37 @@ describe('Bigtable', () => {
       assert.strictEqual(metadata.name, backupNameFromCluster);
       assert.deepStrictEqual(backup.expireDate, updateExpireTime);
     });
+
+    it.only('should get an Iam Policy for the backup', async () => {
+      const policyProperties = ['version', 'bindings', 'etag'];
+      const [policy] = await BACKUP.getIamPolicy();
+      console.log(`policy => \n${JSON.stringify(policy)}`);
```

Comment:
I don't think we should leave any logging in the tests.

Reply:
Removed, thanks.